test/e2e/storage/drivers/in_tree.go

```diff
@@ -1429,6 +1438,7 @@ func InitAzureDiskDriver() testsuites.TestDriver {
 				// Azure supports volume limits, but the test creates large
 				// number of volumes and times out test suites.
 				testsuites.CapVolumeLimits: false,
+				testsuites.CapTopology:     true,
```

Comment:
should there be `TopologyKeys:         []string{v1.LabelZoneFailureDomain}` on Azure?

Reply:
Added.